logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationMapper.kt

```diff
@@ -96,6 +109,152 @@ internal class ConversationMapperImpl(
         accessRole = daoModel.accessRole.map { it.toDAO() }
     )
 
+    @Suppress("ComplexMethod", "LongMethod")
+    override fun fromDaoModel(daoModel: ConversationViewEntity): ConversationDetails = with(daoModel) {
+        when (type) {
+
+            ConversationEntity.Type.SELF -> {
+                ConversationDetails.Self(
+                    Conversation(
+                        id = idMapper.fromDaoModel(id),
+                        name = name,
+                        type = type.fromDaoModelToType(),
+                        teamId = teamId?.let { TeamId(it) },
+                        protocol = protocolInfoMapper.fromEntity(protocolInfo),
+                        mutedStatus = conversationStatusMapper.fromMutedStatusDaoModel(mutedStatus),
+                        removedBy = null, // todo: ask how to calculate?
+                        creatorId = PlainId(""), // deprecated
```

Comment:
If `creatorId` is marked as deprecated, why don't we just delete it instead of keep using it?

Reply:
it will affect everywhere! I'll do it in next PR